components/input-number/InputNumber.razor.cs

```diff
@@ -63,68 +159,122 @@ private void SetClass()
                 .Add(PrefixCls)
                 .If($"{PrefixCls}-lg", () => Size == InputSize.Large)
                 .If($"{PrefixCls}-sm", () => Size == InputSize.Small)
+                .If($"{PrefixCls}-focused", () => _focused)
                 .If($"{PrefixCls}-disabled", () => this.Disabled);
         }
 
         protected override void OnParametersSet()
         {
             base.OnParametersSet();
-
             SetClass();
         }
 
         private void Increase()
         {
-            OnInput(new ChangeEventArgs() { Value = Value + Step });
+            _inputString = _increaseFunc(Value, Step).ToString();
+            ConvertNumber(_inputString);
```

Comment:
Why did you convert twice? Maybe we can call `_roundFunc` directly.

Reply:
OK